hhs_hosp/delphi_hhs/run.py

```diff
@@ -75,8 +75,11 @@ def run_module():
     dfs = []
     for r in date_range:
         response = Epidata.covid_hosp(request_all_states, r)
-        if response['result'] != 1:
+        # The last date range might only have recent days that don't have any data, so don't error.
+        if response["result"] != 1 and r != date_range[-1]:
             raise Exception(f"Bad result from Epidata: {response['message']}")
+        if response["result"] == -2 and r == date_range[-1]:
```

Comment:
What does -2 correspond to? Maybe add a comment.

Reply:
no results, good idea on the comment
```suggestion
        if response["result"] == -2 and r == date_range[-1]:  # -2 code means no results
```